src/attributes.md

```diff
@@ -204,6 +204,8 @@ which can be used to control type layout.
   object file that this item's contents will be placed into.
 - `no_mangle` - on any item, do not apply the standard name mangling. Set the
   symbol for this item to its identifier.
+- [`used`] - on statics, this forces the compiler to keep the variable in the
```

Comment:
I don't like adding more attributes to the misc. attributes section. Could we rename Misc. attributes to "Object file attributes"? We could throw `no_main`, `no_start` and `windows_subsystem` in that heading too possibly (I really don't know much about object files...so taking a guess here). Where `global_allocator` goes, I don't know. Perchance just make it its own section?

Reply:
I'm looking at regrouping and updating the attributes on the attributes page.  I'd like to do that in a separate PR, if that's OK?  I've opened issue #534 to discuss this further, I'd be happy if you could leave some feedback since you've done most of the initial work on this.